sdk/core/azure-core/src/tracing/tracing.cpp

```diff
@@ -30,6 +30,13 @@ namespace Azure { namespace Core { namespace Tracing { namespace _internal {
 
   using Azure::Core::Context;
 
+  std::shared_ptr<TracerProviderImpl> TracerImplFromTracer(
```

Comment:
I don't remember why, but we don't have free-standing functions in the SDK, we put them all in "static classes".
I would ask to change that, but so far this is the current state of things, if I am not mistaking.

Reply:
So be it. Annoying, but if those are the rules, they're the rules.